app/controllers/password_resets_controller.rb

```diff
@@ -22,10 +22,12 @@ def create
   end
 
   def update
-    if @user.update_attributes(user_params)
-      log_in @user
+    update_password = UpdatePassword.call(user_params: user_params, user: @user)
+
+    if update_password.success?
+      log_in update_password.user
```

Comment:
@Enriikke the `log_in` method here and in a couple other spots is provided by our `sessions_helper.rb`. Moving `log_in` to an interactor causes issues. It seems that we can't find the `session` hash to set our `user_id` if the call to `log_in` is moved outside of the controller. We would probably want this call in our interactor though, yes?